.github/workflows/publish-canvas.yml

```diff
@@ -16,35 +16,29 @@ jobs:
         uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
 
       - name: Install Python 3.9
-        # very carefully use the tag when testing with act-cli on localhost, commit the hash version
-        # uses: actions/setup-python@v2
```

Comment:
Not useful comments since we're not using the `act-cli` tool (which sadly didn't work very well on Windows).